react-relay/index.d.ts

```diff
@@ -128,7 +146,13 @@ declare module "react-relay" {
     }
 
     interface RelayProp {
-        variables: any
-        setVariables(variables: Object, onReadyStateChange?: OnReadyStateChange): void
+        route: { name: string; }; // incomplete, also has params and queries
+        variables: any;
+        pendingVariables?: any;
+        setVariables(variables: Object, onReadyStateChange?: OnReadyStateChange): void;
+        forceFetch(variables: Object, onReadyStateChange?: OnReadyStateChange): void;
+        hasOptimisticUpdate(record?: any): boolean;
+        getPendingTransactions(record?: any): RelayMutationTransaction[];
+        commitUpdate?: (mutation: Mutation<any,any>, callbacks?: StoreUpdateCallbacks<any>) => any;
```

Comment:
Why is this typed as being optional? I think [it will always exist](https://github.com/facebook/relay/blob/9e1a1309420f684aab49cdbe98074094236feab2/src/container/RelayContainer.js#L161), but I haven’t used any Relay mutations yet so not sure what happens at runtime.